src/hsqldb19b3/org/hsqldb_voltpatches/SetFunction.java

```diff
@@ -547,7 +547,24 @@ static Type getType(int setType, Type type) {
             case OpTypes.PERCENTILE_75 :
             case OpTypes.PERCENTILE_95 :
             case OpTypes.PERCENTILE_99 :
-                return Type.SQL_DOUBLE;
+                switch (dataType) {
+                case Types.TINYINT :
```

Comment:
missing indentation of nested cases

Reply:
If you look further up the source file, this appears to be VoltDB's style (not to indent `case` statements within `switch` statements). I prefer the style you're describing, but I'm trying to keep with the same style that VoltDB is using here.

Reply:
lol…actually, I take it back. They're using different styles at different points in the same file. 🤷‍♂️

Reply:
👍 makes sense - sorry missed that. github being helpful and collapsing "unrelated blocks"

Reply:
the fact that they do it _anywhere_, to me, moves this suggestion in to the "why bother" category..